### ISSUE TITLE
question ,why can't we access 1/16 and 1/32 directly

### ISSUE THREAD

**Issue by najlitarvan** [tsunamayo/Starship-EVO#6143]
we can access said grids via copy paste, but it feels like weird when i have 1/8 size selected and have grid precision set to quarter grid, that i am stuck to 1/8, wouldn't it make more sense for the block to move on the 1/32 placement grid? why isn't that a thing if i may ask

**Comment by CoasterBlock** [comment]
IIrc, 1/8th is actually the base grid the game is working of off. So a ship that is 1000m long, will be on a grid with 8x of that resolution to be able to accomodate the small bricks. There is some arguments going on right now to make some logic components like buttons, switches and similar items 1/8th compatible. A switch currently is 25x25cm, which can be a little large for small cockpits. 

TLDR: We cant go smaller than 1/8th, because 1/8th is the base grid the world is divided into. And if youre building to large, you quickly reach the limit for decimal imprecission. I ran into that issue with a few of my ships. Collision boxes become imprecise after roughly 20ish km. and unrealiable after 35km. If we where to subdivide the world grid further, this issue would be much more obvious.

**Comment by CoasterBlock** [comment]
IIrc, 1/8th is actually the base grid the game is working of off. So a ship that is 1000m long, will be on a grid with 8x of that resolution to be able to accomodate the small bricks. There is some arguments going on right now to make some logic components like buttons, switches and similar items 1/8th compatible. A switch currently is 25x25cm, which can be a little large for small cockpits. 

TLDR: We cant go smaller than 1/8th, because 1/8th is the base grid the world is divided into. And if youre building to large, you quickly reach the limit for decimal imprecission. I ran into that issue with a few of my ships. Collision boxes become imprecise after roughly 20ish km. and unrealiable after 35km. If we where to subdivide the world grid further, this issue would be much more obvious.

**Comment by ProPeach** [comment]
So the feature that would fix this for you @najlitarvan is the Quarter Grid snapping option in the F1 menu working when using 1/8th and 1/4th bricks?
It is pretty weird that option effectively doesn't do anything when using those smaller scales, I remember there being some reason for it discussed back when neovox was first implemented but can't remember the exact reason

**Comment by najlitarvan** [comment]
> So the feature that would fix this for you @najlitarvan is the Quarter Grid snapping option in the F1 menu working when using 1/8th and 1/4th bricks? It is pretty weird that option effectively doesn't do anything when using those smaller scales, I remember there being some reason for it discussed back when neovox was first implemented but can't remember the exact reason. It's odd having to resort to using thin tiles or the copy paste tool to achieve the smaller offsets when you can just use a convenient setting when using 1m bricks.
ya, screwed up quarter grid and half grid when it comes to 1/8 and 1/4 is my issue

**Comment by najlitarvan** [comment]
@CoasterBlock i am not talking block sizes, i am talking placement precision, though i do wish we could have some kind of button grid to use as smaller buttons, but not what i am after atm
